src/components/Modals/SeeAllTransactionModal/SeeAllTransactionModal.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
+import { t } from 'decentraland-dapps/dist/modules/translation/utils'
+import { Close } from 'decentraland-ui'
+import { ModalProps } from 'decentraland-dapps/dist/providers/ModalProvider/ModalProvider.types'
+import Modal from 'decentraland-dapps/dist/containers/Modal'
+import { AccountTransaction } from '../../HomePage/AccountCard/AccountTransactions/AccountTransaction'
+import './SeeAllTransactionModal.css'
+import { Transaction } from '../../HomePage/HomePage.types'
+
+const SeeAllTransactionModal = ({ name, onClose, metadata }: ModalProps) => {
+  const { transactions } = metadata
+  return (
+    <Modal
+      name={name}
+      className="SeeAllTransactionModal"
+      closeIcon={<Close onClick={onClose} />}
+    >
+      <Modal.Header>{t('see_all_transaction_modal.title')}</Modal.Header>
+      <Modal.Content>
+        {transactions.map(
+          (
+            { type, status, description, amount }: Transaction,
+            index: number
+          ) => (
+            <AccountTransaction
```

Comment:
I'm curious about why this component is not receiving directly a `transaction` object?

Reply:
Because We starting doing only visual stuff with mocked data. But agree with you that this need to be changed with real data, inclusive some types that we use here (AccountTransactionStatus and AccountTransactionType). Can we postpone this change for the real data connection PR?

Reply:
Yes, can you create an issue with it? or adding it to another issue with a reference to this comment so we won't forget?